telemetry/data_window.go

```diff
@@ -80,6 +110,124 @@ var (
 	subWindowsLock    = sync.RWMutex{}
 )
 
+func getSQLSum(sqlTypeDeta *sqlType) int64 {
+	result := sqlTypeDeta.Use
+	result += sqlTypeDeta.Show
+	result += sqlTypeDeta.Begin
+	result += sqlTypeDeta.Commit
+	result += sqlTypeDeta.Rollback
+	result += sqlTypeDeta.Insert
+	result += sqlTypeDeta.Replace
+	result += sqlTypeDeta.Delete
+	result += sqlTypeDeta.Update
+	result += sqlTypeDeta.Select
+	result += sqlTypeDeta.Other
+	return result
+}
+
+func readSQLMetric(timepoint time.Time, SQLresult *sqlUsageData) error {
```

Comment:
Use SQLResult instead of SQLresult

Reply:
Thank you for your commet, I will accept it!